### PR TITLE
Persistent, Inpersistent(temporary) holograms

### PR DESCRIPTION
**Pull request by UsainSrht**

### CODE REVIEW

File: src/main/java/de/oliver/fancyholograms/HologramManagerImpl.java [hunk 1]
```diff
@@ -58,10 +58,21 @@ public final class HologramManagerImpl implements HologramManager {
...
+     * Returns a read-only view of the currently loaded persistent holograms.
+     *
+     * @return A read-only collection of holograms.
+     */
+    @Override
+    public @NotNull
+    @UnmodifiableView Collection<Hologram> getHolograms() {
+        return this.holograms.values().stream().filter(hologram -> hologram.getData().isPersistent()).toList();
+    }
+
```
Comment (OakLoaf) [inline src/main/java/de/oliver/fancyholograms/HologramManagerImpl.java:75]: I think I would personally name this `getPersistentHolograms` and then revert `getAllHolograms` to `getHolograms`.

**Review by OakLoaf — CHANGES_REQUESTED** [review]
This looks really good, would be a helpful feature too!

**Comment by OakLoaf** [comment]
Just thought I'd prompt this thought that I just had - would `HologramManagerImpl#reloadHolograms` cause all non-persistent holograms to be deleted on plugin reload?

**Comment by UsainSrht** [comment]
> Just thought I'd prompt this thought that I just had - would `HologramManagerImpl#reloadHolograms` cause all non-persistent holograms to be deleted on plugin reload?

probably, let me test and resolve it if thats the case

**Comment by OakLoaf** [comment]
This PR will resolve #50

File: src/main/java/de/oliver/fancyholograms/HologramManagerImpl.java (hunk 1 above)
Comment (OakLoaf) [reply src/main/java/de/oliver/fancyholograms/HologramManagerImpl.java:75]: Awesome, I think the only thing left would be to look at those methods names mentioned above

**Comment by OakLoaf** [comment]
Awesome, I think the only thing left would be to look at those methods names mentioned above

**Review by OakLoaf — APPROVED** [review]
This looks good to me!

**Pull request merged**